enterprise/bundle.go

```diff
@@ -26,10 +31,34 @@ func (a *aliasFS) Open(name string) (fs.File, error) {
 	if ok {
 		return a.FS.Open(fileAlias)
 	}
-	return a.FS.Open(name)
+	f, err := a.FS.Open(name)
+	// If the file does not exist, attempt to read it from runfiles.
+	// Otherwise, return the original result.
+	if err != nil && os.IsNotExist(err) {
+		// Fall back to bazel runfiles. Note the limitation here that we can only get
+		// runfiles under the enterprise dir.
+		runfile, err := bazelgo.Runfile(filepath.Join("enterprise", name))
+		if err == nil {
+			return os.Open(runfile)
+		}
+	}
```

Comment:
would it make sense to have a file resolver type util that looks at runfiles or the bundle (from env)? Then we could fix up all callsites that use bundle/runfiles directly to just call that util and get this behavior.

to me it seems confusing to put this here since if you're importing bundle, you expect to be reading from the bundle, not runfiles.

Reply:
Done